pages.zh/linux/grub-install.md

```diff
@@ -0,0 +1,16 @@
+# grub-install
+
+> 安装 GRUB 到设备。
+> 更多信息：<https://www.gnu.org/software/grub/manual/grub/html_node/Installing-GRUB-using-grub_002dinstall.html>。
+
+- 安装 GRUB 到基于 BIOS 的系统：
+
+`grub-install --target={{i386-pc}} {{path/to/device}}`
+
+- 安装 GRUB 到基于 UEFI 的系统：
+
+`grub-install --target={{x86_64-efi}} --efi-directory={{path/to/efi_directory}} --bootloader-id={{GRUB}}`
+
+- 安装预置指定模块的 GRUB ：
```

Comment:
```suggestion
- 安装预置指定模块的 GRUB:
```

Reply:
all line is chinese  endwith a english character `:` , it seem to be strange.

Reply:
If you'd like to know the reasoning behind this, check out #5240. But basically, we use the "style" of the character immediately before it.